src/main/resources/applicationContext.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/context
+        http://www.springframework.org/schema/context/spring-context.xsd"
+       default-init-method="initObject"
+       default-destroy-method="destroyObject"
+       default-lazy-init="true"
+>
+    <bean id="prototypeBean" class="com.homework.PrototypeBean" scope="prototype"
+          init-method="initObjectPrototype"
+          destroy-method="destroyObjectPrototype">
+    </bean>
+    <bean id="singletonBean" class="com.homework.SingletonBean" scope="singleton"
```

Comment:
Scope singleton is the default one. So you could not write scope="singleton". It is redundant. 
Now it is tendentious to use java code configuration, not XML.